ocrd_validator.py

```diff
@@ -205,33 +205,33 @@ def __validate_min_amount_of_tokens(self, line_num, line, min_amount):
         if len(line) < min_amount:
             expected = f"At least {min_amount} tokens."
             got = f"{len(line)} tokens"
-            hint = 'TOKEN_ERROR04: Each line must have a processor call, an input file, and an output file'
+            hint = 'LINE_ERROR_RULE_02: Each line must have a processor call, an input file, and an output file'
             self.__print_syntax_error(line_num, expected=expected, got=got, hint=hint)
             sys.exit(2)
 
     def __validate_QM_tokens(self, line_num, first_qm_token, last_qm_token):
         if not first_qm_token == QM:
-            hint = 'TOKEN_ERROR05: OCR-D commands must start with a quotation mark'
+            hint = 'TLINE_ERROR_RULE_03: OCR-D commands must start with a quotation mark'
             self.__print_syntax_error(line_num, expected=QM, got=first_qm_token, hint=hint)
```

Comment:
The same as above. A separate info variable should be created and passed in the method. The info should report the error and what caused the error. The hint section provides more details on how to fix it.
E.g., `info = "LINE_ERROR_RULE_03: Wrong start of the OCR-D command"`